CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.22.0 / 2023-06-15
+
```

Comment:
```suggestion

* [FIX] Updates to various MIB download locations and parsing
```

Reply:
We generally haven't bothered to mention generator example changes.